examples/explore_browser_use/initial_actions.py

```
@@ -0,0 +1,36 @@
+
+url="https://en.wikipedia.org/wiki/Randomness"
+context="Username : Admin, Password : admin123"
+
+initial_actions = [
+	{'go_to_url': {'url': url, 'new_tab': True}},
+	{'scroll': {'down': True, 'num_pages': 1}},
+]
+
+llm = ChatOpenAI(model="gpt-4.1")
+
+async def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)